test/arbitrage/BancorArbitrage.ts

```diff
@@ -529,6 +522,9 @@ describe('BancorArbitrage', () => {
             const uniswapV3RouterSameOutput = sameOutputExchanges;
             const sushiswapV2RouterSameOutput = sameOutputExchanges;
 
+            // exclude v3 due to new arb contract deployment
+            exchangeIds.splice(exchangeIds.indexOf(ExchangeId.BancorV3), 1);
```

Comment:
why is this needed?

Reply:
We're deploying a new arb contract here with new mock exchanges, to test out the 0 reward and burn amount case.
We deploy a new contract, b/c the exchanges addresses are immutable, but when we deploy a new contract, we can't trade on the same v3 - because the arb contract is immutable in the network. 
So if we want to trade using the v3 network here, we need to make an entirely new deployment (which uses a different address for the arb contract, meaning we can't reuse the createSystem function).